### PR TITLE
Add info to secure MAP in HTTPS

### PR DESCRIPTION
**Pull request by smau-centreon**
## Description

Add info to secure MAP in HTTPS

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

**Review by nmagniezCentreon — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/secure-your-map-platform.md [hunk 1]
```diff
@@ -51,6 +55,12 @@ centreon-map.keystore-pass=xxx
 >
 > Les utilisateurs devront ouvrir l'URL :
 >
+> - MAP :
+> ```shell
+> https://<MAP_IP>:9443/centreon-studio/api/beta/actuator/health.
```
Comment (Jordan91) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/secure-your-map-platform.md:60]: https://IP-MAP:9443/centreon-map/api/beta/actuator/health

Comment (Jordan91) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/secure-your-map-platform.md:60]: it's not centreon-studio in path for map engine

File: versioned_docs/version-22.04/graph-views/secure-your-map-platform.md [hunk 2]
```diff
@@ -60,6 +64,12 @@ centreon-map.keystore-pass=xxx
 >
 > Users will have to open the URL:
 >
+> - MAP:
+> ```shell
+> https://<MAP_IP>:9443/centreon-studio/api/beta/actuator/health.
```
Comment (Jordan91) [inline versioned_docs/version-22.04/graph-views/secure-your-map-platform.md:69]: https://IP-MAP:9443/centreon-map/api/beta/actuator/health

File: versioned_docs/version-22.10/graph-views/secure-your-map-platform.md [hunk 3]
```diff
@@ -60,6 +64,12 @@ centreon-map.keystore-pass=xxx
 >
 > Users will have to open the URL:
 >
+> - MAP:
+> ```shell
+> https://<MAP_IP>:9443/centreon-studio/api/beta/actuator/health.
```
Comment (Jordan91) [inline versioned_docs/version-22.10/graph-views/secure-your-map-platform.md:69]: https://IP-MAP:9443/centreon-map/api/beta/actuator/health

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/secure-your-map-platform.md [hunk 4]
```diff
@@ -51,6 +55,12 @@ centreon-map.keystore-pass=xxx
 >
 > Les utilisateurs devront ouvrir l'URL :
 >
+> - MAP :
+> ```shell
+> https://<MAP_IP>:9443/centreon-studio/api/beta/actuator/health.
```
Comment (Jordan91) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/secure-your-map-platform.md:60]: https://IP-MAP:9443/centreon-map/api/beta/actuator/health

**Review by Jordan91 — COMMENTED** [review]
little fix and it's ok for me

**Review by Jordan91 — APPROVED** [review]

**Pull request merged**